cli/tests/containerize.bats

```diff
@@ -400,15 +400,16 @@ EOF
     hello.pkg-path = "hello"
 
     [containerize.config]
-    Cmd = [ "hello" ]
+    cmd = [ "hello" ]
 EOF
   )"
 
   echo "$MANIFEST_CONTENTS" | _FLOX_USE_CATALOG_MOCK="$GENERATED_DATA/resolve/hello.json" "$FLOX_BIN" edit -f -
 
-  TAG="Cmd-runs-in-activation"
+  TAG="cmd-runs-in-activation"
 
-  bash -c "FLOX_CONTAINERIZE_FLAKE_REF_OR_REV=main $FLOX_BIN containerize --tag $TAG --runtime podman" 3>&- # TODO: why close FD 3?
+  # TODO: Reset REF_OR_REV to `main` after merging.
+  bash -c "FLOX_CONTAINERIZE_FLAKE_REF_OR_REV=dcarley/containerize-manifest-format $FLOX_BIN containerize --tag $TAG --runtime podman" 3>&- # TODO: why close FD 3?
```

Comment:
NB: To revert after merging.

#2502 and the preceding test not running on Mac, made it take several CI cycles to get the tests passing.